files/docker-compose.yml

```
@@ -0,0 +1,104 @@
+---
+version: "3.7"
+
+secrets:
+  admiral_yml:
+    file: ./secrets/admiral.yml
+  mongo_root_passwd_txt:
+    file: ./secrets/mongo-root-passwd.txt
+  redis_conf:
+    file: ./secrets/redis.conf
+  sslmate_api_key_txt:
+    file: ./secrets/sslmate-api-key.txt
+
+x-admiral-template: &admiral-template
+  build:
+    context: .
+    dockerfile: Dockerfile-admiral
+  deploy:
+    mode: replicated
+    replicas: 5
+  environment:
+    ADMIRAL_CONFIG_FILE: "/run/secrets/admiral.yml"
+    ADMIRAL_CONFIG_SECTION: dev-mode
+    ADMIRAL_WORKER_NAME: dev
+  image: admiral
+  init: true
+  secrets:
+    - source: admiral_yml
+      target: admiral.yml
+    - source: sslmate_api_key_txt
+      target: sslmate-api-key.txt
+  volumes:  # map for development only, comment out otherwise
+    - ./src/admiral:/usr/src/admiral/admiral
+
+services:
+  celery-flower:
+    environment:
+      # TODO: https://github.com/cisagov/admiral/issues/19
+      # move into the secrets raft
+      CELERY_BROKER_URL: "redis://:fruitcake@redis:6379/0"
+    # TODO: https://github.com/cisagov/admiral/issues/18
+    # replace this outdated image
+    image: crgwbr/docker-celery-flower
+    ports:
+      - "5555:5555"
+
+  cert-worker:
+    <<: *admiral-template
+    environment:
+      ADMIRAL_CONFIG_SECTION: cert-worker
+      ADMIRAL_WORKER_NAME: cert
+
+  mongo:
+    environment:
+      MONGO_INITDB_DATABASE: certs
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD_FILE: /run/secrets/mongo_root_passwd_txt
+    image: mongo:4.2
+    restart: always
+    secrets:
+      - source: mongo_root_passwd_txt
+    volumes:
+      - ./init/mongo-init.js:/docker-entrypoint-initdb.d/mongo-init.js:ro
+
+  mongo-express:
+    environment:
+      ME_CONFIG_MONGODB_SERVER: mongo
+      ME_CONFIG_MONGODB_ADMINUSERNAME: root
+      ME_CONFIG_MONGODB_ADMINPASSWORD: example
+    image: mongo-express:0.54
+    ports:
+      - "8083:8081"
+    restart: always
+
+  redis:
+    command:
+      - "redis-server"
+      - "/run/secrets/redis_conf"
+    image: redis:7.0-alpine
```

I see that [`redis:7.2.1-alpine`](https://hub.docker.com/layers/library/redis/7.2.1-alpine/images/sha256-ff84de0300ff103207d78e777fb007c397974a000ef861c91664bfbbefa56b83?context=explore) is out.  You should test with it and if all is well, use it here:
```suggestion
    image: redis:7.2.1-alpine
```